lib/datasource/github-releases/index.ts

```diff
@@ -13,6 +11,8 @@ type GithubRelease = {
   published_at: string;
 };
 
+const http = new GithubHttp(id);
```

Comment:
The first param here is meant to be `hostType`, which is maybe best set to be `github`.